src/content/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more.mdx

```diff
@@ -88,18 +88,57 @@ Notes on these settings:
 
 For how to set up SCIM, see [Automated user management](/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign).
 
-## Authentication: username/password versus SAML SSO  [#authentication]
+### User type upgrade requests [#user-upgrade]
+
+In the **Source of users** UI, you have two options for how your users' [user type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type) is managed:
+
+* **Manage user type in New Relic**: This is the default option. It allows you to manage your users' user type from New Relic.
+* **Manage user type with SCIM**: Enabling this means that **you can no longer manage user type from New Relic**. You'd only be able to change and manage it from your identity provider.
+
+More on these two options:
+
+<CollapserGroup>
+  <Collapser
+    id="manage-user-type-new-relic"
+    title="Manage user type in New Relic (default)"
+  >
+    The default way to manage your users' user type is from the New Relic UI. This is the option you'd use if you aren't [managing user type via SCIM](#manage-user-type-scim). The management options governed by this include:
+
+    * The ability to change user type in the [**User management** UI](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks#where).
+    * The ability to [manage how user type upgrade requests are handled](#user-upgrade). 
+  </Collapser>
+
+  <Collapser
+    id="manage-user-type-scim"
+    title="Manage user type with SCIM"
+  >
+    Our [SCIM API](/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim#manage-user-type) lets you manage user type from your identity provider instead of in the New Relic user management UI. **When you enable this, you can no longer change or manage your users' user type from New Relic.**
+
+    To enable this setting, your authentication domain **Source of users** must have [SCIM enabled](#source-users).
+
+    Currently, we only have instructions for [Okta](/docs/accounts/accounts/automated-user-management/okta-scimsso-application-configuration). For other identity provider services, you can configure your identity provider to send us user type using our [SCIM API user type specifications](/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim/#manage-user-type).
```

Comment:
"send us user type" reads a little weirdly to me. Maybe "send us user type information" or something like that?

Reply:
yeah, weird stuff